RELEASE-PROCESS.md

```diff
@@ -0,0 +1,53 @@
+# Release Process
+
+External facing [Traffic Director](https://cloud.google.com/traffic-director)
+documentation for specific features tend to use the wallet example to showcase
+the feature better. Here is one such [example][wallet-example-in-td-docs] where
+a specific branch of the wallet example is used.
+
+[wallet-example-in-td-docs]:
+https://cloud.google.com/traffic-director/docs/proxyless-configure-advanced-traffic-management#verify-routing-configuration
+
+# When to make a new release
+Whenever we have a new iteration of the public documentation, referencing the
+wallet example to showcase new set of functionality, we should consider making a
+new release. Public documentation for the new set of functionality would end up
+referencing the newly created version. This would ensure that existing
+documentation wouldn't need any changes, and any changes we make to the examples
+to support new functionality would not end up breaking old documentation
+inadvertently.
+
+# How to make a new release
+Once all changes are made to examples to support the new functionality, we
+should create a new branch to represent the new release.
+
+This approach has the advantage that any bug fixes that need to be made on a
+release branch do not require a patch release. The public documentation will
```

Comment:
The public documentation will always refer to `HEAD` of these branches. This approach has the advantage that any bug fixes that need to be made on a branch do not require any update to the documentation.